docs/configfile.rst

```diff
@@ -0,0 +1,122 @@
+
+.. _chi2lim:
+
+CHI2LIM: The lower and upper limits of :math:`\chi^2` values from the processed AMISR data file that should be included in the velocity reconstruction.  The :math:`\chi^2` parameter indicates goodness of fit of the original ISR fitting procedure.  Large values indicate excessively large errors while small values suggest "overfitting" and that the values should not be trusted even if errors are small.  Data points with :math:`\chi^2` values outside this range will be filtered out.
+
+.. _goodfitcode:
+
+GOODFITCODE: The list of fit codes (assigned by the AMISR fitter) that indicate a successful fit.  Generally, fit codes :math:`\le 0` or :math:`\ge 5` mean the fit failed and that point should be excluded.
+
+.. _binvert:
+
+BINVERT: The list of vertices that define each bin.  These should be given in magnetic coordinates.  See the :ref:`Velocity Bins <bins>` page for more details about how to specify these.
+
+.. _outalt:
+
+OUTALT: The list of altitudes (in kilometers) geodetic components of the output velocity and electric field should be calculated at.  The output files contain the Apex components, which are constant along magnetic field lines, but for convenience, geodetic components are also included for a discrete grid defined by the locations of the bin centers and this output altitude array.
+
+.. _marprot:
+
+MARPROT: The rotation coordinates for the MARP coordinate system.
+
+.. _upb_beamcode:
+
+UPB_BEAMCODE: (Optional) Beam code for the beam pointing directly up the magnetic field.  Some radar modes may not include an "up-B" beam.  The RISR-N field-of-view is tilted too far North for any beam to be oriented along the magnetic field, so config files written for RISR-N experiments should NEVER include this field.
+
+.. _ionup:
+
+IONUP: (Optional) The method used to calculate and correct for any ion upflow component of the velocity.
+
+.. _use_beams:
+
+USE_BEAMS: (Optional) A list of specific beam codes that should be used for the velocity reconstruction.  If this field is omitted, all available beams from the experiment will be used.
+
+.. _inttime:
+
+INTTIME: (Optional) Post-integration period (in seconds) that should be used to reconstruct vectors for a single time stamp in the output file.  Because the input processed data has discrete time stamps (usually on a cadence greater than one minute), the output file will typically not have exactly this resolution unless an integration time is selected that is an exact multiple of the cadence of the input data file.  Instead, the code with post-integrate the smallest number of input time stamps that cover a period greater than the specified integration time.  If this parameter is omitted, the native times of the input file are used (no post-integration).
```

Comment:
```suggestion
INTTIME: (Optional) Post-integration period (in seconds) that should be used to reconstruct vectors for a single time stamp in the output file.  Because the input processed data has discrete time stamps (usually on a cadence greater than one minute), the output file will typically not have exactly this resolution unless an integration time is selected that is an exact multiple of the cadence of the input data file.  Instead, the code will post-integrate the smallest number of input time stamps that cover a period greater than or equal to the specified integration time.  If this parameter is omitted, the native times of the input file are used (no post-integration).
```